addons/mail/controllers/discuss.py

```
@@ -285,3 +285,65 @@ def message_get_suggested_recipients(self, model, res_ids):
         except Exception:
             return {}
         return records._message_get_suggested_recipients()
+
+    # --------------------------------------------------------------------------
+    # RTC API
+    # --------------------------------------------------------------------------
+
+    @http.route('/mail/rtc/session/notify_call_members', type="json", auth="public")
+    def session_call_notify(self, sender, targets=None, content=None):
+        if request.env.user._is_public():
+            guest = request.env['mail.guest']._get_guest_from_request(request)
+            if guest:
+                guest.env['mail.channel.rtc.session'].sudo().browse(int(sender))._notify_peers(targets, content)
+                return
```

need to raise if guest is not found

Applies to all routes.